examples/game/lib/image_map.dart

```diff
@@ -15,7 +15,7 @@ class ImageMap {
   }
 
   Future<Image> _loadImage(String url) async {
-    Image image = await _bundle.loadImage(url);
+    Image image = await _bundle.loadImage(url).first;
```

Comment:
This must be a pretty low-level function to want to grab at a specific frame in the image?